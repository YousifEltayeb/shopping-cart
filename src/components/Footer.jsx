import styled from "styled-components";
const FooterStyle = styled.footer`
    margin-top: auto;
    background-color: #171616;
`;
const Footer = () => {
    return (
        <FooterStyle>
            Made With ❤️ By{" "}
            <a href="https://github.com/YousifTheKind" target="_blank">
                Yousif Eltayeb
            </a>
            <p style={{ fontSize: "12px" }}>
                As part of the{" "}
                <a href="https://theodinproject.com/" target="_blank">
                    The Odin Project
                </a>{" "}
            </p>
        </FooterStyle>
    );
};

export default Footer;
