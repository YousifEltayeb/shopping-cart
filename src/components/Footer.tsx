import styled from "styled-components";
const FooterStyle = styled.footer`
  margin-top: auto;
  background-color: #171616;
`;
const Footer = () => {
  return (
    <FooterStyle>
      Created By{" "}
      <a href="https://github.com/YousifTheKind" target="_blank">
        Yousif Eltayeb
      </a>
    </FooterStyle>
  );
};

export default Footer;
