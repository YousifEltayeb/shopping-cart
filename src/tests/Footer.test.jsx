import { describe, it, expect } from "vitest";
import { render } from "@testing-library/react";
import Footer from "../components/Footer";
import { MemoryRouter } from "react-router";
describe("Footer", () => {
  it("Footer is rendered", () => {
    const { container } = render(
      <MemoryRouter>
        <Footer />
      </MemoryRouter>,
    );
    expect(container).toMatchSnapshot();
  });
});
