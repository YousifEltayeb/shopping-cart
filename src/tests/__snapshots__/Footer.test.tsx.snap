// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Footer > Footer is rendered 1`] = `
<div>
  <footer
    class="sc-FEMpB bGcbJv"
  >
    Made With ❤️ By
     
    <a
      href="https://github.com/YousifTheKind"
      target="_blank"
    >
      Yousif Eltayeb
    </a>
    <p
      style="font-size: 12px;"
    >
      As part of the
       
      <a
        href="https://theodinproject.com/"
        target="_blank"
      >
        The Odin Project
      </a>
       
    </p>
  </footer>
</div>
`;
